src/dmd/expression.d

```diff
@@ -1531,6 +1537,47 @@ StringExp semanticString(Scope *sc, Expression exp, const char* s)
     return se;
 }
 
+/***************************************************
+ * Given an Expression, find the variable it really is.
+ * for example, `a[index]` is really `a`, and `s.f` is really `s`.
```

Comment:
Capital `f` after period. I also recommend having adding a newline before this line. That would allow the first line to be used as a summary.